parts/swarmagentresourcesvmss.t

```diff
@@ -173,6 +175,26 @@
 {{end}}
             }
           }
+{{if IsRHEL}}
+          ,"extensionProfile": {
+            "extensions": [
+              {
+                "name": "configure{{.Name}}",
+                "properties": {
+                  "publisher": "Microsoft.Azure.Extensions",
+                  "settings": {
+                    "commandToExecute": "[variables('agentCustomScript')]",
+                    "fileUris": [
+                      "[concat('https://raw.githubusercontent.com/tomconte/acs-engine/tco-rhel-swarmmode/parts/', variables('configureClusterScriptFile'))]"
```

Comment:
@tomconte this is looking good overall, how can we clean up these static links to your github fork?

Reply:
I think the simplest is to just point to the master branch of the acs-engine repo for now... The only risk there is that if the configuration script is broken for some reason on master, deployments will fail. And at some point we might be able to point to a stable release branch instead?

Reply:
For the extensions work with @robbagby we had an approach where it would look to github *unless* a parameter/property was overridden. This allowed you to override in dev (and even to resolve from a storage account while you're working on it rather than needing to push to github to test it)

Could something similar work here?

Reply:
Makes sense! I will check it out.